standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -1122,6 +1154,49 @@ public void abortTxns(AbortTxnsRequest rqst) throws MetaException {
     }
   }
 
+  private void markDbAsReplIncompatible(Connection dbConn, String database) throws SQLException, MetaException {
```

Comment:
We needn't to have almost a copy of updateReplId(). If you need similar code for both externalize that part. I was also wondering if we would ever need and update.